services/ux-backend/handlers/onboardingtokens/handler.go

```diff
@@ -32,9 +33,9 @@ func HandleMessage(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours
 }
 
 func handlePost(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours int) {
-	var storageQuotaInGiB *uint
-	// When ContentLength is 0 that means request body is empty and
-	// storage quota is unlimited
+
+	var roleSpec services.OnboardingSubjectRoleSpec
+	roleSpec.Role = services.ClientRole
```

Comment:
setting fields at multiple places forces us to look for where all these are set, better to move this to line 59 only where you are setting the options?

Reply:
Since we have only one role up to this point, I am always setting the role as clientRole to avoid further changes to console. With 4.18, we will explicitly set the role while calling this endpoint